docs/docs.scss

```diff
@@ -13,6 +13,16 @@ $snippet-padding: 12px;
 $snippet-content-background: #f2f0eb;
 $snippet-border: 2px solid #CCC;
 
+.icon-block {
+    width: 75px;
+    height: 75px;
+    display: inline-block;
+    background-color: #eee;
+    line-height: 75px;
+    text-align: center;
+    border-radius: 3px;
```

Comment:
nit: probably good idea to stick with alphabetical ordering of properties. I know this in docs and hence not covered by our standards but probably just a good idea to be consistent.